login-workflow/CHANGELOG.md

```diff
@@ -5,11 +5,12 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## v4.0.4 (Unreleased)
+## v4.1.0 (Unreleased)
 
 ### Added
 
--   Error Manager for ChangePasswordDialog ([#612](https://github.com/etn-ccis/blui-react-workflows/issues/612))
+-   Reusable okta login screen.
+-   Error Manager for ChangePasswordDialog ([#612](https://github.com/etn-ccis/blui-react-workflows/issues/612)).
 
```

Comment:
@KomalGDhere-eaton , can you please make sure all issues that we fixed are logged here.

Reply:
sure